### ISSUE TITLE
is

### ISSUE THREAD

**Issue by tanjing9406** [vasturiano/react-force-graph#338]
**Describe the bug**
Is there an event handle for mousemove? I want to draw link when mousemove, but onMouseMove prop in ForceGraph2D is not working

**Comment by vasturiano** [comment]
@sorenhoyer yes that would be the right thing to do. This component does not have a `onMouseMove` method, nor would it make sense to have one. Those are event methods that really belong to lower level HTML elements, so it would make the most sense to overlay an element to act as a mouse event trap. That would also give you the most flexibility for what you want to do with it.
